### PR TITLE
Update the docker custom build

### PR DESCRIPTION
**Pull request by saurabharch**
Update syntax command for docker COPY package.json and package*.json  and n8n packages in the root working directory folder

### CODE REVIEW

File: docker/images/n8n-custom/Dockerfile [hunk 1]
```diff
@@ -11,14 +11,14 @@ RUN apk --update add --virtual build-dependencies python3 build-base ca-certific
...
-COPY packages/nodes-base/ ./packages/nodes-base/
-COPY packages/workflow/ ./packages/workflow/
+COPY package.json package*.json .
+COPY packages/* ./packages
+#COPY packages/cli/ ./packages/cli/
+#COPY packages/core/ ./packages/core/
+#COPY packages/design-system/ ./packages/design-system/
+#COPY packages/editor-ui/ ./packages/editor-ui/
+#COPY packages/nodes-base/ ./packages/nodes-base/
+#COPY packages/workflow/ ./packages/workflow/
```
Comment (ivov) [inline docker/images/n8n-custom/Dockerfile:21]: This is now including `packages/node-dev`

Comment (saurabharch) [reply docker/images/n8n-custom/Dockerfile:21]: please merge this request with https://github.com/n8n-io/n8n/pull/3846

File: docker/images/n8n-custom/Dockerfile [hunk 2]
```diff
@@ -10,15 +10,16 @@ RUN apk --update add --virtual build-dependencies python3 build-base ca-certific
...
-COPY turbo.json .
-COPY package.json .
-COPY package-lock.json .
-COPY packages/cli/ ./packages/cli/
-COPY packages/core/ ./packages/core/
-COPY packages/design-system/ ./packages/design-system/
-COPY packages/editor-ui/ ./packages/editor-ui/
-COPY packages/nodes-base/ ./packages/nodes-base/
-COPY packages/workflow/ ./packages/workflow/
+COPY lerna.json .
```
Comment (netroy) [inline docker/images/n8n-custom/Dockerfile:13]: there is no `lerna.json` in this repo anymore

**Comment by netroy** [comment]
This got addressed in https://github.com/n8n-io/n8n/pull/3973

**Comment by netroy** [comment]
@saurabharch Thanks for your contribution :heart_decoration:

**Pull request closed without merging**